modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/FieldFeedback.tsx

```diff
@@ -15,30 +15,25 @@ export function FieldFeedback({
 	warningMessage,
 	...otherProps
 }: IProps) {
-	if (!errorMessage && !helpMessage && !warningMessage) {
-		return null;
-	}
+	const message =
+	warningMessage && !errorMessage ? warningMessage : errorMessage;
+
+	const symbol =
+		warningMessage && !errorMessage ? 'warning-full' : 'exclamation-full';
```

Comment:
Same here `warningMessage ? 'warning-full' : 'exclamation-full'`

Reply:
This means that if there's an error message, it should be shown even if there is also a warning message. The warningMessage should only be shown if there is not an error message. Is that what's intended? 

Since this is the same check, it would be better to set default values and reset them inside an `if` block.

Reply:
That's correct @ethib137. 
I had to make it that way because  JAWS wasn't relating the `id` on `feedbackField` to  `aria=describedby` on `ClayInput`.

Reply:
@ethib137, I'll make the changes you requested.
@pat270, I will work on the tests and snapshots as well.

Thanks guys!